### ISSUE TITLE
Box decoration is not working

### ISSUE THREAD

**Issue by idkashutosh** [Norbert515/flutter_ide#30]

**Comment by Norbert515** [comment]
Indeed, I've been working on implementing it, but it is pretty tricky to get right.
I wanted to write a generic implementation of properties which rely on object subclasses but it's not finished as of now
